waku/common/wakubridge.nim

```diff
@@ -23,63 +23,91 @@ import
   # Common cli config
   ./config_bridge
 
+logScope:
+  topics = "wakubridge"
+
+##################
+# Default values #
+##################
+
 const
+  defaultBridgeTopic* = "/waku/2/default-bridge/proto"
```

Comment:
Could you elaborate on rationale of this? I'm still a bit jetlagged so it might be obvious, I don't see a mention of a dedicated bridge topic in the spec though https://specs.vac.dev/specs/waku/v2/waku-bridge

Reply:
Well, since there is no Waku v1 equivalent to a PubSub topic we need something to default to when translating from v1 to v2. The question then is, when we do the inverse, if the bridge should translate _from_ all subscribed PubSub topics or whether we should also provide some default topic that is used for communication on both v1 and v2 networks. 
In both cases the usual `defaultTopic` would suffice, but I've decided to keep the bridged messages in a separate `defaultBridgeTopic` to avoid noise - at least while the `bridge` is still being developed and proper duplicate filtering is still lacking.

The spec does not mention PubSub topics, but the wording in #205 made me think that this may have been the original intention. Perhaps @kdeme can comment?

Reply:
> decided to keep the bridged messages in a separate defaultBridgeTopic to avoid noise - at least while the bridge is still being developed and proper duplicate filtering is still lacking.

makes sense